pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraMergeCandidates.scala

```diff
@@ -68,11 +68,9 @@ object SierraMergeCandidates
 
   /** We can merge a single-page Miro and Sierra work if:
     *
-    *   - The Sierra work has type "Picture" or "Digital images"
-    *   - There's exactly one Miro ID in MARC tag 962 subfield $u
-    *     (if there's more than one Miro ID, we can't do a merge).
-    *   - There's exactly one item on the Sierra record (if there's more
-    *     than one item, we don't know where to put the Miro location).
+    *   - The Sierra work has type "Picture", "Digital images", or "3D Object"
+    *   - There's exactly one Miro ID in MARC tag 962$u or 089$a
+    *     (if there's more than one Miro ID in either, we can't do a merge)
```

Comment:
I don't think this is in the scope of this PR, but just a note that I think (??) we intend to relax this and allow merging multiple Miros into 1 Sierra? I know that the cases where we don't do that are a definite pain point at the moment (eg https://wellcomecollection.org/works?query=borax%20pet)

Reply:
> we can't do a merge

As in the merger - why not?

Reply:
Not sure where this came from, happy to dig it out further as another PR.
https://github.com/wellcomecollection/catalogue/issues/536